src/fancyline/tty/vt100.cr

```diff
@@ -90,6 +90,14 @@ class Fancyline
         @io.print "\e[2J\e[H"
       end
 
+			def switch_to_alternate_screen
```

Comment:
The methods you added should also be part of the Tty class as `abstract def`s.  There, document the methods as appropriate.  (Rule of thumb: Every public API should have full docs coverage)

Reply:
Didn't notice that class 😅 No probs, I'll add the docs once we're settled on the API.